Roster

sofiaroz opened the pull request:
- must reseed

emilyzwang commented on client/src/components/Profile.js:
```diff
@@ -63,83 +68,124 @@ const Profile = () => {
+              </Row>{' '}
+              <Row type="flex">
+              <Col>
+                <p>
+                  <span className="profileItem" id="semestersAttendedBubble">
+                    SEMESTERS
+                  </span>{' '}
+                  {/* {displayStudentSemesters} */}
+                </p>
```
do we want to display semesters here?

emilyzwang commented on client/src/components/RosterCard.js:
```diff
@@ -41,99 +78,261 @@ const RosterCard = props => {
+            </span>{' '}
+            {email}
+          </p>
+          <p>
```
+ fav candy

emilyzwang commented on client/src/components/RosterCard.js:
```diff
@@ -1,37 +1,74 @@
   const editStudentProfile = () => {
-    if (editedNotes.length >= 255) {
+    let updateNotes = editedNotes;
+    if (!updateNotes) {
+      updateNotes = notes;
+    }
```
issues if user is trying to delete notes or if `notes` is null

emilyzwang commented on controllers/rosters_controller.js:
```diff
@@ -10,29 +10,59 @@ const getUsersBySite = async (req, res) => {
   }
 };
+
+const getUserSemester = async (req, res) => {
```
do we use this?

sofiaroz replied:
- got rid of this after collapsing semesters column

sofiaroz commented:
- collapsed semesters column from user_site (previously, user_semester_site) table into studentSemesters in user table
- updated migration files
- renamed user_semester_site table appropriately to user_site (note: file names have not been updated, to update file names after merging all PRs to prevent conflicts)
